tests/core/scene/a-scene.test.js

```diff
@@ -388,6 +388,70 @@ suite('a-scene (without renderer)', function () {
     });
   });
 
+  suite('resize', function () {
+    test('resize not in vr updates renderer size', function () {
```

Comment:
difficult to read. May be better like this? `resize renderer when not in vr mode`